client/src/pages/Achieves/utils.ts

```diff
@@ -10,7 +10,7 @@ import { GameAttempt, User } from "../../interfaces/User";
 import { AchievesId } from "./types";
 
 export const checkIsAchieveOpen = (user: User | null, achieveId: AchievesId) => {
-  if (user) {
+  if (user && user.statistics.length !== 0) {
```

Comment:
You may use shorter statements. Because if length = 0 it will be a falsy. So you can use just (user && user.statistics.length).